nestkernel/rng_manager.h

```diff
@@ -44,7 +44,7 @@ namespace nest
 
 class RNGManager : public ManagerInterface
 {
-  friend void MPIManager::set_num_rec_processes( int, bool ); // create_rngs_
+  friend void MPIManager::initialize(); // calls create_rngs_
```

Comment:
Might not be required. See [previous comment](https://github.com/nest/nest-simulator/pull/653/files?diff=unified#r101513246)